libs/full/actions/tests/unit/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ endif()
 set(set_thread_state_PARAMETERS THREADS_PER_LOCALITY 4)
 set(thread_stacksize_PARAMETERS LOCALITIES 2)
 set(thread_affinity_PARAMETERS THREADS_PER_LOCALITY 4 RUN_SERIAL)
-set(thread_affinity_LIBRARIES DEPENDENCIES PRIVATE Hwloc::hwloc)
```

Comment:
Was this intentional?

Reply:
Intentional, but not a must. Since HPX already depends on hwloc I figured we don't really need to put this here explicitly. But having it there wouldn't be wrong either.